node/src/types/block.rs

```diff
@@ -378,20 +378,14 @@ impl FinalizedBlock {
         self.proposer.clone()
     }
 
-    /// Returns an iterator over all deploy and transfer hashes.
-    pub(crate) fn deploys_and_transfers_iter(
-        &self,
-    ) -> impl Iterator<Item = DeployOrTransferHash> + '_ {
-        self.deploy_hashes
-            .iter()
-            .copied()
-            .map(DeployOrTransferHash::Deploy)
-            .chain(
-                self.transfer_hashes
-                    .iter()
-                    .copied()
-                    .map(DeployOrTransferHash::Transfer),
-            )
+    /// Returns the deploy hashes for the finalized block.
```

Comment:
Maybe to clarify, since "deploy" is a bit ambiguous:

```suggestion
    /// Returns the deploy hashes for the finalized block, excluding transfers.
```

Reply:
I've tried to clarify the documentation in https://github.com/casper-network/casper-node/pull/1712/commits/b1f535c93aa991626a75eb3a0750c8d4d8fdc99b